packages/openid4vc/src/openid4vc-issuer/repository/OpenId4VcIssuanceSessionRecord.ts

```diff
@@ -97,6 +98,18 @@ export type DefaultOpenId4VcIssuanceSessionRecordTags = {
   presentationAuthSession?: string
 }
 
+export interface OpenId4VcIssuanceSessionRecordTransactionData {
+  // The expected number of credentials that will be issued in this transaction
+  numberOfCredentials: number
+
+  // The format of the credential request that was used to create the issuance session
+  requestFormat?: CredentialRequestFormatSpecific
+
+  // The credential configuration and its id
+  credentialConfigurationId?: string
```

Comment:
I think we should store the configuration id for the transaction (and so make this required), so we know which configuration id the deferred issuance is assocaited with.

Reply:
I don't think it is strictly necessary, no. I can remove it. We fetch it on demand, right, if it's not given?

Reply:
Because we need the credential configuration to subsequently call `getSignedCredentials`